packages/cli/src/commands/list.js

```diff
@@ -30,7 +50,17 @@ export const list = async ({ directory, follow, json }) =>
     } else {
       const petNames = await E(agent).list();
       for await (const petName of petNames) {
-        console.log(petName);
+        if (verbose) {
+          let spaces = 20 - `${petName}`.length;
+          if (spaces < 2) {
+            spaces = 2;
+          }
+          const spacer = '                    '.substring(0, spaces);
```

Comment:
I think I was overindexing on performance, which in this case really isn't significant enough to matter.  The motivation behind the original formulation was that the string of spaces was static, rather than rebuilding the string each pass through the loop.  But if you sign up for the cost of using `repeat` then I think it makes even more sense to instead just say:
```
const spacer = ' '.repeat(spaces);
```
which upon looking at it seems preferable to either alternative because the underlying intent is much clearer.